e2e/aks_model.go

```diff
@@ -19,6 +20,68 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/privatedns/armprivatedns"
 )
 
+// getLatestGAKubernetesVersion returns the highest GA Kubernetes version for the given location.
+func getLatestGAKubernetesVersion(location string, t *testing.T) (string, error) {
+	versions, err := config.Azure.AKS.ListKubernetesVersions(context.Background(), location, nil)
+	if err != nil {
+		return "", fmt.Errorf("failed to list Kubernetes versions: %w", err)
+	}
+	if len(versions.Values) == 0 {
+		return "", fmt.Errorf("no Kubernetes versions available")
+	}
+
+	var latestPatchVersion string
+	// Iterate through the available versions to find the latest GA version
+	t.Logf("Available Kubernetes versions for location %s:", location)
+	for _, k8sVersion := range versions.Values {
+		if k8sVersion == nil {
+			continue
+		}
+		t.Logf("- %s", *k8sVersion.Version)
+
+		// Skip preview versions
+		if k8sVersion.IsPreview != nil && *k8sVersion.IsPreview {
+			t.Log(" - - is in preview, skipping")
```

Comment:
missing the '%s' here?

Reply:
no it under the version 
- 1.27.1
- - is in preview, skipping
- 1.27.2
- -  is in preview, skipping